packages/kbn-router-to-openapispec/src/oas_converter/kbn_config_schema/post_process_mutations/mutations/utils.ts

```diff
@@ -35,6 +35,13 @@ export const processDeprecated = (schema: OpenAPIV3.SchemaObject): void => {
   }
 };
 
+export const processDiscontinued = (schema: OpenAPIV3.SchemaObject): void => {
+  if (metaFields.META_FIELD_X_OAS_DISCONTINUED in schema) {
+    schema.discontinued = schema['x-oas-discontinued'] as string;
```

Comment:
nit: let's use the constant?
```suggestion
    schema.discontinued = schema[metaFields.META_FIELD_X_OAS_DISCONTINUED] as string;
```